example/helloWorld/src/helloworld.cpp

```diff
@@ -0,0 +1,190 @@
+     * - AccGpuCudaRt
+     * - AccCpuThreads
+     * - AccCpuFibers
+     * - AccCpuOmp2Threads
+     * - AccCpuOmp2Blocks
+     * - AccCpuOmp4
+     * - AccCpuSerial
+     *
+     * Each accelerator has strengths and weaknesses. Therefore,
+     * they need to be choosen carefully depending on the actual
+     * use case. Furthermore, some accelerators only support a
+     * particular workdiv, but workdiv can also be generated
+     * automatically.
+     *
+     */
+    using Dim     = alpaka::dim::DimInt<3>;
+    using Size    = std::size_t;
+    using Host    = alpaka::acc::AccCpuSerial<Dim, Size>;
+    using Acc     = alpaka::acc::AccCpuSerial<Dim, Size>;
+    using Stream  = alpaka::stream::StreamCpuSync;
+    using DevAcc  = alpaka::dev::Dev<Acc>;
+    using DevHost = alpaka::dev::Dev<Host>;
+    using WorkDiv = alpaka::workdiv::WorkDivMembers<Dim, Size>;
+
+
+    /**
+     * Get the first devices
+     *
+     * The accelerator only defines how something should be
+     * parallized, but a device is the real entity which will
+     * run the parallel programm. The device can be choosen
+     * by id (0 to the number of devices minus 1) or you
+     * can also retrieve all devices in a vector (getDevs()).
+     * In this example the first devices is choosen.
+     *
+     */
+    DevAcc  devAcc  (alpaka::dev::DevMan<Acc>::getDevByIdx(0));
+    DevHost devHost (alpaka::dev::DevMan<Host>::getDevByIdx(0));
+
+    
+    /**
+     * Create a stream to the accelerator device
+     *
+     * A stream can be interpreted as the work queue
+     * of a particular device. Streams are filled with
+     * execution units and alpaka takes care that these
+     * units will be executed. Streams are provided in
+     * async and sync variants. 
+     * The example stream is a sync stream to a cpu device, 
+     * but it also exists an async stream for this 
+     * device (StreamCpuAsync).
+     */
+    Stream  stream  (devAcc);
+
+
+    /**
+     * Init workdiv
+     *
+     * The workdiv defines how many instances of a
+     * kernel can be spawned in parallel. How many kernels
+     * are really spawned finally depends on the choosen
+     * accelerator e.g. the sequential accelerator will
+     * only spawn one kernel at a time, whereas a CUDA
+     * accelerator can spawn hundreds of kernels at the
+     * same time.
+     *
+     * The workdiv is divided in three levels of parallelization:
+     * - Grid     : consists of blocks
+     * - Block    : consists of threads
+     * - Elements : consists of elements
+     *
+     * Threads in the same grid can access the same global memory,
+     * while threads in the same block can access the same shared
+     * memory. Elements are supposed to be used for vectorization.
+     * Thus, a thread can process data element size wise with its
+     * vector processing unit.
+     *
+     */
+    alpaka::Vec<Dim, Size> const elem (static_cast<Size>(1),
```

Comment:
I am not sure. I think we don't expect to much if we provide a three dimensional kernel here. Because multi dimensional kernel is a fundamental feature of alpaka.